ui/pages/Search/components/VariantSearchFormPanelConfigs.jsx

```diff
@@ -0,0 +1,285 @@
+import React from 'react'
```

Comment:
everything in this file is copied from `VariantSearchFormContent` or a constants file without edits